src/comp/comp.py

```diff
@@ -25,47 +27,56 @@ def __repr__(self):
 # whose name starts with 'D':
 print("Starts with D:")
 a = []
+[a.append((str(v)[:-5][8:])) for v in humans if "Human: D" in str(v) ]
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
 b = []
+[b.append((str(v)[:-5][8:])) for v in humans if "e," in str(v) ]
 print(b)
```

Comment:
Solid work. Remember when using List comprehensions we set them equal to a variable form the start. Here that would be b = [LC without `append()`].